crates/wasmcloud-host/src/manifest.rs

```diff
@@ -125,6 +127,9 @@ pub(crate) async fn generate_provider_start_messages(
                 .and_then(|bytes| ProviderArchive::try_load(&bytes))
                 .and_then(|par| NativeCapability::from_archive(&par, cap.link_name.clone()))
             {
+                if let Err(e) = crate::capability::native_host::write_provider_to_disk(&prov) {
+                    error!("Could not cache provider to disk: {}", e);
+                }
```

Comment:
Just to confirm, this is re-writing the par that was sourced from disk to the cache location, and this is done because otherwise we would just be loading the par from the cache even if it had changed?

Reply:
Yes, that's it.

Reply:
Oops, no, I read that backwards. The original version loads the cache even if the par has changed, this version updates the cache with the par contents if the par is defined in manifest as a local file.

Reply:
I may have written that backwards 😅, we're on the same page!